__tests__/app.test.js

```diff
@@ -22,6 +23,22 @@ describe('app testing', () => {
                 expect(body.msg).toBe('Invalid input!');
             })
         })
+        test('Returns a 400 status and an error message when an invalid ID has been entered', () => {
+            return request(app)
+            .get('/api/articles/two')
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Invalid article request!');
+            })
+        })
+        test('Returns a 404 status and an error message when the requested article does not exist', () => {
+            return request(app)
+            .get('/api/articles/99999')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Requested article not found!');
+            })
+        })
```

Comment:
These tests are great but i would recommend having these in the describe block for get article by id as these error tests are for that end point specifically.